addons/web_settings_dashboard/models/res_users.py

```diff
@@ -16,10 +16,12 @@ def web_dashboard_create_users(self, emails):
             user.active = True
 
         new_emails = set(emails) - set(deactivated_users.mapped('email'))
+        return self.web_dashboard_create_new_users(list(new_emails))
 
+    @api.model
+    def web_dashboard_create_new_users(self, new_emails):
```

Comment:
Can't you override web_dashboard_create_users in auth signup, handle emails, remove emails to which we will resend activation password and resume it ? It seems that way we can avoid adding a new method and still achieve the result. Could you give it a try ?

By the way don't forget to udpate commit message according to guidelines.